ddtrace/contrib/consul/patch.py

```diff
@@ -0,0 +1,52 @@
+import consul
+
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+
+from ...ext import AppTypes
+from ...pin import Pin
+from ...utils.wrappers import unwrap as _u
+
+
+_KV_FUNCS = ['Consul.KV.put', 'Consul.KV.get', 'Consul.KV.delete']
+
+
+def patch():
+    if getattr(consul, '__datadog_patch', False):
+        return
+    setattr(consul, '__datadog_patch', True)
+
+    pin = Pin(service='consul', app='consul', app_type=AppTypes.cache)
+    pin.onto(consul.Consul)
+    pin.onto(consul.Consul.KV)
+
+    for f_name in _KV_FUNCS:
+        _w('consul', f_name, wrap_function(f_name))
+
+
+def unpatch():
+    if not getattr(consul, '__datadog_patch', False):
+        return
+    setattr(consul, '__datadog_patch', False)
+
+    for f_name in _KV_FUNCS:
+        name = f_name.split('.')[-1]
+        _u(consul.Consul.KV, name)
+
+
+def wrap_function(name):
+    def wrapping_func(original_func, instance, args, kwargs):
```

Comment:
```suggestion
    def trace_func(wrapped, instance, args, kwargs):
```

Minor suggestion for function name and argument name to keep consistent with codebase.